docs/en/query_language/functions/other_functions.md

```diff
@@ -178,6 +178,39 @@ Accepts Float32 and Float64 and returns UInt8 equal to 1 if the argument is not
 
 Accepts Float32 and Float64 and returns UInt8 equal to 1 if the argument is infinite, otherwise 0. Note that 0 is returned for a NaN.
 
+## ifNotFinite {#ifnotfinite}
+
+Accepts Float32 and Float64. The function has the [ternary operator](https://clickhouse.tech/docs/en/query_language/functions/conditional_functions/#ternary-operator) structure: `ifNotFinite(x, y) = isFinite(x) ? x : y`. It returns `x` if `x` if finite, and `y` if `x` is not finite. 
```

Comment:
The function does something between input and output, this is the point of this description. For parameters and returned values you have two sections below.

What the function does? That is a question.

The function checks whether `x` is a finite value. That is the answer.